packages/blockchain-api/package.json

```diff
@@ -14,7 +14,8 @@
     "build": "tsc -p .",
     "clean": "rm -rf dist coverage",
     "gcp-build": "npm run build",
-    "deploy": "./deploy.sh"
+    "deploy": "./deploy.sh",
+    "postinstall": "test -f src/secrets.json || echo \"{}\" > src/secrets.json"
```

Comment:
The way to do this is contemplated here: https://github.com/celo-org/celo-monorepo/blob/master/scripts/create_key_templates.sh (although it's not well documented)

Reply:
Great thanks, gonna change it to this then!

Reply:
🙌